components/pluralrules/src/data.rs

```diff
@@ -122,6 +122,7 @@ pub enum RulesSelector {
 
 impl RulesSelector {
     pub fn select(&self, operands: &PluralOperands) -> PluralCategory {
+        println!("select: {:?}", operands);
```

Comment:
^

Reply:
Yeah, I should not forget the printf debugging leftovers.